packages/venia-concept/_buildpack/create.js

```diff
@@ -91,6 +91,7 @@ function createProjectFromVenia({ fs, tasks, options }) {
                     pkg.scripts[name] = toPackageScript(scriptsToInsert[name]);
                 });
 
+                // eslint-disable-next-line
```

Comment:
Would be helpful for readers if we explicitly named the rule we were disabling when we do this.

```
// eslint-disable-next-line no-alert
```

Reply:
Good suggestion! I got disable happy :P I'll do this.